gamecenter/settings.py

```diff
@@ -5,9 +5,11 @@
 
 
 class Config(object):
-    SECRET_KEY = os_env.get('GAMECENTER_SECRET', 'secret-key')  # TODO: Change me
-    APP_DIR = os.path.abspath(os.path.dirname(__file__))  # This directory
+    # TODO: Change me
```

Comment:
be more descriptive with this TODO, idk; what 'change me' means. Why change what, and how?